test/functional/test-custom-element.js

```diff
@@ -268,17 +268,21 @@ describe('CustomElement', () => {
 
     container.appendChild(element);
     element.attachedCallback();
+    expect(element).to.have.class('-amp-element');
```

Comment:
Could you please add one more test where you attach second time and make sure that `amp-notbuilt` is not added again?